### ISSUE TITLE
how to reproduce the classification results

### ISSUE THREAD

**Issue by M-leng** [qq456cvb/Point-Transformers#23]
I trained the Hengshuang's model. Under the default setting, I only got 89.7177 (instance average Acc). How to reproduce the result "91.7" listed in the table.
![image](https://user-images.githubusercontent.com/15032403/150080844-8ea363a5-b5eb-4238-ade9-8f91271cbb5b.png)

**Comment by qq456cvb** [comment]
Hi, Leng, you may look at the issue #13, and use the hyperparameters there. See if you can reproduce the results.

**Comment by yuchenlichuck** [comment]
Hi, Leng, I found that I used the pre-trained model to achieve 91.7%

**Comment by agoodge** [comment]
> Hi, Leng, I found that I used the pre-trained model to achieve 91.7%

Where is the pre-trained model available?

**Comment by cygao1215** [comment]
如何可视化分类结果

**Comment by qq456cvb** [comment]
The table numbers are the best instance accuracy over a full 200-epoch run, and run-to-run variance of 1 to 2% is normal from seed, augmentation, and LR. Make sure you train the full schedule rather than stopping early, and keep the default augmentation. As the README notes, the Hengshuang/Nico LRs probably aren't fully tuned, so a bit of tuning can help close the gap.
